### PR TITLE
Update in-meeting app readme

### PR DESCRIPTION
**Pull request by JerryYangKai**

### CODE REVIEW

**Comment by JerryYangKai** [comment]
@zyxiaoyuer Please help review, thanks

File: hello-world-in-meeting/README.md [hunk 1]
```diff
@@ -1,49 +1,59 @@
...
 > App Caching was configured in this sample to reduce the reload time of your app in a meeting. To learn about limitations and available scopes, please check [Enable app caching for your tab app](https://learn.microsoft.com/en-us/microsoftteams/platform/apps-in-teams-meetings/app-caching-for-your-tab-app).
 
-Microsoft Teams supports the ability to run web-based UI inside "custom tabs" that users can install either for just themselves (personal tabs) or within a team or group chat context. Please be advised that mgt-teamsfx-provider library in this app is currently in preview stage, please expect breaking changes in the future release.
+## This sample illustrates
 
-## Prerequisites
+- Use Teams Toolkit to create a Teams meeting app.
```
Comment (summzhan) [inline hello-world-in-meeting/README.md:11]: How to use...

File: hello-world-in-meeting/README.md [hunk 2]
```diff
@@ -1,49 +1,59 @@
...
+- Use Teams SDK to get context data in Teams app.
```
Comment (summzhan) [inline hello-world-in-meeting/README.md:12]: TeamsFx SDK？

File: hello-world-in-meeting/README.md [hunk 3]
```diff
@@ -1,49 +1,59 @@
...
-- In Visual Studio Code
+1. Open the `Run and Debug Activity` panel from sidebar, or use short key Ctrl+Shift+D.
+1. Select `Launch Remote (Edge)` or `Launch Remote (Chrome)` in the launch configuration (a dropdown selection in the upper-left corner).
+1. Press the `Start Debugging` (small green arrow) button to launch your app, the Teams web client will be automatically opened in your browser, where you will see your app running remotely from Azure.
+
+## Version History
+
+| Date         | Author | Comments      |
+| ------------ | ------ | ------------- |
+| Mar 16, 2023 | Kai    | first version |
```
Comment (summzhan) [inline hello-world-in-meeting/README.md:55]: I remember in V4 we already have this in-meeting sample.
So please add one line history for TTK V4.

**Review by summzhan — CHANGES_REQUESTED** [review]
please resolve the comments in line

File: hello-world-in-meeting/README.md (hunk 1 above)
Comment (JerryYangKai) [reply hello-world-in-meeting/README.md:11]: Fixed

File: hello-world-in-meeting/README.md (hunk 2 above)
Comment (JerryYangKai) [reply hello-world-in-meeting/README.md:12]: Teams client lib, in this sample we have not used teamsfx sdk

File: hello-world-in-meeting/README.md (hunk 3 above)
Comment (JerryYangKai) [reply hello-world-in-meeting/README.md:55]: Fixed

**Comment by JerryYangKai** [comment]
> please resolve the comments in line

@summzhan  Thanks for reviewing this, I have pushed a new commit and fix the readme

**Review by summzhan — APPROVED** [review]

**Pull request merged**